src/System.Security.Cryptography.Cng/src/project.json

```diff
@@ -25,6 +26,11 @@
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6.1": "1.0.1"
       }
+    },
+    "net463": {
+      "dependencies": {
+        "Microsoft.TargetingPack.NETFramework.v4.6.2": "1.0.1"
```

Comment:
Yes. Eric is working on the 4.6.3 pack which currently I need to check in the test projects, unless I fall back to getting P2P working for those. I will create an issue